profile.html

```diff
@@ -61,6 +61,10 @@ <h4 class="col-xs-8 col-xs-offset-2 col-md-6">About</h4>
       <textarea class="form-control" rows="3" placeholder="What's your story? What poems do you like?" required></textarea>
     </div>
   </div>
+  <div class="save-btn col-xs-8 col-xs-offset-2 ">
```

Comment:
I recommend adding a comment describing what is being saved, or modifying the 'save-btn' class name to include what being saved, i.e., save-bio-btn.